src/uu/tail/src/args.rs

```diff
@@ -148,16 +149,13 @@ impl Settings {
         settings.retry =
             matches.get_flag(options::RETRY) || matches.get_flag(options::FOLLOW_RETRY);
 
-        if let Some(s) = matches.get_one::<String>(options::SLEEP_INT) {
-            settings.sleep_sec = match s.parse::<f32>() {
-                Ok(s) => Duration::from_secs_f32(s),
-                Err(_) => {
-                    return Err(UUsageError::new(
-                        1,
-                        format!("invalid number of seconds: {}", s.quote()),
-                    ))
-                }
-            }
+        if let Some(source) = matches.get_one::<String>(options::SLEEP_INT) {
+            settings.sleep_sec =
```

Comment:
Could you add a quick comment here explaining what the advantage of `fundu` is over `try_from_f64_secs`?

Reply:
sure, no problem but it'll take until tomorrow

Reply:
done